docs/content/toolbar.md

```diff
@@ -57,20 +57,35 @@ Opens the [Inspector](https://docs.improbable.io/reference/13.1/shared/glossary#
 
 The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal GDK** > **Toolbar**.
 
- ![Ttoolbar](../assets/screen_grabs/toolbar/toolbar_settings.png)
+ ![Toolbar](../assets/screen_grabs/toolbar/toolbar_settings.png)
 
+* **Configuration**
 
-* **Project Root Folder**</br>
-By default, this points to the root folder of your SpatialOS Unreal GDK project. If you're using a non-standard structure, you'll need to set this yourself.
+    * **Project Root Folder**</br>
+    If you're using a non-standard structure, you'll need to set this yourself. By default, this is empty. If this is left empty, then the it will default to `<ProjectFileLocation>/../spatial`.
 
-* **Launch Configuration**</br>
-The [launch configuration file](https://docs.improbable.io/reference/13.1/shared/reference/file-formats/launch-config) (SpatialOS documentation) to use when running `spatial local launch` using the `Launch` button.
+    * **Launch Configuration**</br>
+    The [launch configuration file](https://docs.improbable.io/reference/13.1/shared/reference/file-formats/launch-config) (SpatialOS documentation) to use when running `spatial local launch` using the `Launch` button.
 
-* **Stop on Exit**</br>
-If enabled, shuts down running deployments when you close the Unreal Editor.
+    * **Stop on Exit**</br>
+    If enabled, shuts down running deployments when you close the Unreal Editor.
 
-* **Snapshot path**</br>
-Path to your SpatialOS Unreal GDK snapshot.
+    * **Snapshot path**</br>
+    Path to your SpatialOS Unreal GDK snapshot. **Note* If this is left empty, then the it will default to `<ProjectFileLocation>/../spatial/snapshots`.
```

Comment:
Formatting won't work. Missing colon. "then the it" is a typo. Turned it to active voice and present tense rather than future prediction "will". Present tense is stronger.
```
**Note:** If you leave this left empty, then the it defaults to `<ProjectFileLocation>/../spatial/snapshots`.
```